examples/CloudRetro/README.md

```diff
@@ -0,0 +1,509 @@
+
+* <YOUR_C_CLUSTER_IP>  - The in-cluster IP of your coordinator, the workers should connect to this.
+* <YOUR_C_EXTERNAL_IP> - The exposed IP of your coordinator, clients should use this to access the service at the sepcified port (8000 in this example).
+* <YOUR_W_EXTERNAL_IP> - The exposed IP of the collection of your workers.
+
+To use these values, we need to edit the configmap weve made:
+
+```console
+kubectl edit configmap -n cloudretro cloudretro-config
+```
+
+Then, we should be presented an ugly looking config. Finding the proper attributes and overwriting their values is the next step.
+You can find the nice looking version of it [here](https://github.com/giongto35/cloud-game/blob/master/configs/config.yaml), and their line numbers presented as ln.
+
+* (ln. 41) coordinator -> server -> address: Should be the port youve used during the coordinator deployment (8000 in this example)
+* (ln. 59) worker -> network -> coordinatorAddress: This should be your <YOUR_C_CLUSTER_IP>:PORT, where the port is the value youve used at the coordinator deployment (8000 in this example)
+* (ln. 65) worker -> network -> publicAddress: This should be your <YOUR_W_EXTERNAL_IP>
+* (ln. 65) worker -> server -> address: Should be the TCP port youve used during the worker deployment (9000 in this example)
+
+After overwriting and saving these values, you will need to restart the deployments for this changes to take effect:
+
+```console
+kubectl rollout restart deployment -n cloudretro worker-deployment
+kubectl rollout restart deployment -n cloudretro coordinator-deployment
+```
+
+Now, if you connect to the coodinator HTTPs, you should be welcomed with the same screen. But if you check your worker list with the little w button on the top-left corner of the console, you should see your workers.
+Congratulations, you`ve (almost) set up the CloudRetro on your cluster!
+
+If you check your browsers console, you could be seeing an ICE-candidate exchange, which ultimately results in fail. That is because if we take a look at the worker-pod log the client is trying to build a connection with, only generates a host type Ice candidate, which is unreachable for the client due to the massive amounts of NATs its probably behind of. This is when STUNner comes into picture.
+
+
+
+### Installing STUNner
+
+STUNner is a Kubernetes ingress gateway for WebRTC, it exposes a standards-compliant STUN/TURN gateway for clients to access your virtualized WebRTC infrastructure running in Kubernetes.
+We will be using STUNner to forward our WebRTC media stream.
+
+First, we will deploy STUNner as a turn server, and configure it, then create a configurable Gateway for it, so we can forward an UDP Route through that. And all this inside the cluster. Marvelous.
+
+First of all, create a namespace for stunner:
+
+```console
+kubectl create namespace stunner
+```
+
+#### Installing STUNner gateway operator
+
+This will be a bit tricky and unlogical, but it wil be fixed asap.
```

Comment:
We now have a helm based process for that, please use the docs from here: https://github.com/l7mp/stunner-helm. Also, please try out the helm charts and bug-report if something doesn't work